safebrowser.go

```diff
@@ -338,6 +340,8 @@ func NewSafeBrowser(conf Config) (*SafeBrowser, error) {
 		}
 	}
 
+	sb.nextlookuptime = time.Unix(0,0)
```

Comment:
mmm...I am not sure what "separate pull request" mean, should I discard this "MinWaitDuration“ commit?

I am a c/c++ engineer and new to go, I am working on a email antispam project that scan 10M emails every day, so I have to sent as less request as possible. Base on the doc of Google, a client should respect server's MinWaitDuration respond, that's why I make the change. Can "a separate pull request" work it around?
Thank you again